lb_content_resolver/unresolved_recording.py

```diff
@@ -42,13 +42,33 @@ def add(self, recording_mbids):
             of times it has been unresolved.
         """
 
-        query = """INSERT INTO unresolved_recording (recording_mbid, last_updated, lookup_count)
-                        VALUES (?, ?, 1)
-         ON CONFLICT DO UPDATE SET lookup_count = EXCLUDED.lookup_count + 1"""
+        recording_mbids = list(set(recording_mbids))
```

Comment:
```suggestion
        recording_mbids = tuple(set(recording_mbids))
```

Together with the change below.

No need to convert to `list` then to `tuple` as it is never modified.
`set()` is only needed if we want to be sure elements are unique.